controllers/mlb_player.go

```diff
@@ -75,3 +78,50 @@ func (ctr *MLBPlayerController) GetMLBPlayerByID(w http.ResponseWriter, r *http.
 
 	json.NewEncoder(w).Encode(player)
 }
+
+func (ctr *MLBPlayerController) GetMLBPlayerDesired(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	filterType := r.FormValue("type")
+
+	if v, ok := allowedTypeFilters[filterType]; !ok || !v {
+		w.WriteHeader(http.StatusBadRequest)
+		json.NewEncoder(w).Encode(errorMessage{
+			Message: "type param value is not allowed",
+		})
+
+		return
+	}
+	itemsperworkers, err := strconv.Atoi(r.FormValue("items_per_workers"))
```

Comment:
you can probably do an extra check to make sure you don't invalid integer values, like less or equal than 0. Same for the other params.